Assets/MixedRealityToolkit.Providers/WindowsVoiceInput/WindowsDictationInputProvider.cs

```diff
@@ -41,6 +41,124 @@ public class WindowsDictationInputProvider : BaseInputDeviceManager, IMixedReali
         /// <inheritdoc />
         public bool IsListening { get; private set; } = false;
 
+        /// <inheritdoc />
+        public async void StartRecording(GameObject listener, float initialSilenceTimeout = 5, float autoSilenceTimeout = 20, int recordingTime = 10, string micDeviceName = "")
+        {
+            await StartRecordingAsync(listener, initialSilenceTimeout, autoSilenceTimeout, recordingTime, micDeviceName);
+        }
+
+        /// <inheritdoc />
+        public async void StopRecording()
+        {
+            await StopRecordingAsync();
+        }
+
+#if UNITY_STANDALONE_WIN || UNITY_WSA || UNITY_EDITOR_WIN
+        /// <inheritdoc />
```

Comment:
I would scope these tighter, inside the function; at least to avoid two function declarations, as when the signature changes we will hit this issue again if the second is forgotten to be updated.

Reply:
That was how I had it originally, to match the SpeechInputProvider, but I couldn't think of another way to resolve this warning without just defining a non-async version in the other function declaration: 
![image](https://user-images.githubusercontent.com/3580640/56763565-9c15c980-6757-11e9-97f9-f41b5d558bc2.png)

Reply:
Ah, I suppose I could define a no-op await or something.

Reply:
Updated to tighten the scope.